README.md

```diff
@@ -1,6 +1,35 @@
-# curriculum-cr-training-mobile-menu
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
+# Personal Portfolio
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+This website was designed with the tools below:
```

Comment:
- [ ] Your REAME looks good, but it requires improvements. It will be nice to list all the technologies and tools used to build that website